adoc/Private-Registry-Installation.adoc

```diff
@@ -316,6 +316,9 @@ internalTLS:
 ====
 Internal TLS support does not yet cover the internal database and Redis services.
 ====
++
+If {spr} is deployed in k3s, note that unmodified traefik (the default ingress controller bundled with k3s) will not work with automatically generated certificates.
+For such setup, It is necessary to add `insecureSkipVerify = true` option for traefik. Learn how to modify traefik settings in link:https://rancher.com/docs/k3s/latest/en/helm/#customizing-packaged-components-with-helmchartconfig[upstream documentation].
```

Comment:
Does this needs to be on another paragraph? I would put this together with the previous paragraph.

Mabye `...it is necessary to configure traefik  to not verify the backend SSL certificate (insecureSkipVerify = true)...`